pinot-broker/src/test/java/org/apache/pinot/broker/requesthandler/MultiStageQueryThrottlerTest.java

```diff
@@ -307,9 +309,75 @@ public void testDisabledToEnabledTransitionDisallowed()
     }
   }
 
+  @Test
+  public void testCalculateMaxServerQueryThreads() {
+    // Neither config is set, both use defaults
+    when(_helixAdmin.getConfig(any(),
+        eq(Collections.singletonList(CommonConstants.Helix.CONFIG_OF_MULTI_STAGE_ENGINE_MAX_SERVER_QUERY_THREADS)))
+    ).thenReturn(Map.of(CommonConstants.Helix.CONFIG_OF_MULTI_STAGE_ENGINE_MAX_SERVER_QUERY_THREADS, "-1"));
+
+    PinotConfiguration emptyConfig = new PinotConfiguration(); // No MSE_MAX_SERVER_QUERY_THREADS set
+    _multiStageQueryThrottler = new MultiStageQueryThrottler(emptyConfig);
+    _multiStageQueryThrottler.init(_helixManager);
+
+    Assert.assertEquals(0, _multiStageQueryThrottler.calculateMaxServerQueryThreads(emptyConfig));
```

Comment:
Switch the 2 arguments. First one is `actual` and second one is `expected`. Same for other asserts